dev/NumberBox/NumberBox.xaml

```diff
@@ -39,6 +39,7 @@
                                         <Setter Target="DownSpinButton.Visibility" Value="Visible" />
                                         <Setter Target="UpSpinButton.Visibility" Value="Visible" />
                                         <Setter Target="InputEater.Visibility" Value="Visible" />
+                                        <Setter Target="InputBox.MinWidth" Value="{ThemeResource NumberBoxMinWidth}" />
```

Comment:
I think if you create a NumberBox with sufficient height, wouldn't the delete button just get wider and in some edge cases still exhibit the issue with spin buttons being on top of the delete button?

Reply:
The overlapping issue is fixed by moving UpdateSpinButtonPlacement() to OnLoaded, since previously InputBox wasn't getting the UpdateVisualState calls properly. This change will give deleteButton a right margin of 72 and fix the overlap.

The MinWidth is just to give the numberBox to a clickable input area when in Inline mode.

Reply:
Ah I see, got it.